src/tests/challenge-manager.test.ts

```diff
@@ -45,6 +94,21 @@ test('manual bisection', () => {
     )
   ).toThrowError('Consensus witness cannot be the last stored state');
 
+  const leafWintness = generateWitness(cm.lastCalldata, 0);
+  const lastNode = leafWintness.nodes.pop() as string;
+  const nonLeafWitness: WitnessProof = {witness: lastNode, nodes: leafWintness.nodes, index: 0};
+
+  expect(() =>
+    cm.split(
+      nonLeafWitness,
+      fingerprints(correctStates, [5, 6]),
+      generateWitness(cm.lastCalldata, 1),
+      challengerId
+    )
+  ).toThrowError(
```

Comment:
nit but I don't love that the "manual bisection test" also incorporates testing invalid splits. I'd personally prefer splitting this into it's own test case.

Reply:
That's a good point! Addressed in f8364e6

Reply:
That's a good point! Addressed in f8364e6